packages/dds/task-manager/src/taskManager.ts

```diff
@@ -57,12 +57,12 @@ const snapshotFileName = "header";
  *
  * ### Usage
  *
- * To volunteer for a task, use the `lockTask()` method.  This returns a Promise that will resolve once the client
- * has acquired exclusive rights to run the task, or reject if the client is removed from the queue without acquiring
- * the rights.
+ * To volunteer for a task, use the `volunteerForTask()` method.  This returns a Promise that will resolve once the
+ * client has acquired exclusive rights to run the task, or reject if the client is removed from the queue without
```

Comment:
My plan is to have it resolve false if the task is already completed since that isn't an "error", but we also still don't have the task lock and we should make that clear. I wasn't sure if I wanted to include changing the return type in this PR since we aren't doing the completed stuff now, but I figured it was a good opportunity since I'm changing the call signature in this PR anyway.